CONTRIBUTING.md

```diff
@@ -86,6 +87,9 @@ Official high quality brand logos and brand colors can usually be found in the f
 1. Wikimedia (which should provide a source)
 1. GitHub repositories
 
+It may be the case that official sources simply do not exist, but an unofficial icon has gained widespread acceptance and popularity. These icons can also be included, after being reviewed and accepted by maintainers on a case-by-base basis. Javascript is the prime example of this situation.
+Notice that an unofficial icon won't replace or supersede an official one, even if it more popular. An unofficial icon will only be accepted if official options do not exist.
```

Comment:
A couple suggestions for brevity:

```suggestion
Notice that an unofficial source will never supersede an official one, even if it is more popular. An unofficial icon will only be accepted if no official option exists.
```

Reply:
"exist" would also be changed to "exists"